addon/adapters/node.js

```diff
@@ -1,8 +1,14 @@
 import ApplicationAdapter from './application';
 
 export default ApplicationAdapter.extend({
-    buildURL() {
+    buildURL(_, __, ___, requestType) {
 	// Embed contributors
-	return `${this._super(...arguments)}?embed=contributors`;
+	var base = this._super(...arguments);
+	if (requestType === 'GET') {
```

Comment:
I was recently looking along parallel lines... does this work? It seems like requestType is the Ember query type, not an HTTP method. See:
https://github.com/emberjs/data/blob/v2.5.1/addon/-private/adapters/build-url-mixin.js#L54